### ISSUE TITLE
NaN when back-propagation

### ISSUE THREAD

**Issue by TuZehai** [mpariente/pytorch_stoi#7]
Hi,

I met an issue when using the torch_stoi in the training. The gradients will become NaN sometimes, and I fixed this by adding an +EPS in line 127 and line 128. I think the sqrt of 0 might lead to a NaN problem.

Thanks

**Comment by mpariente** [comment]
Thanks for reporting this, can you make a PR please?

**Comment by TuZehai** [comment]
First time making a PR, not sure whether it works. Thanks

**Comment by mpariente** [comment]
Well done :tada:
